src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -295,11 +295,30 @@
             "type": "boolean",
             "secret": False,
         },
+        {
+            "id": "request_timeout",
+            "label": gettext_noop("Verify SSL"),
+            "type": "string",
```

Comment:
Shouldn't it be a number?

Reply:
Fixing all, thanks for catching my late night PR blunder

Reply:
This is a float, and float is not an option so string is used.